src/coreComponents/physicsSolvers/multiphysics/SinglePhasePoromechanicsSolver.cpp

```diff
@@ -130,6 +137,12 @@ void SinglePhasePoromechanicsSolver::setupSystem( DomainPartition & domain,
 
 void SinglePhasePoromechanicsSolver::initializePostInitialConditionsPreSubGroups()
 {
+  integer const isFlowThermal = flowSolver()->getReference< integer >( FlowSolverBase::viewKeyStruct::isThermalString() );
```

Comment:
Will it be better to directly give the value of `m_isThermal` flag in flowSolver to `m_isThermal` in this poromechanics solver? Then we don't need to specify two `isThermal`s in xml.

Reply:
Done, I did what you suggested